app/src/main/res/values/strings.xml

```diff
@@ -240,5 +240,11 @@
   <string name="administrator_controls_app_version">App Version</string>
   <string name="administrator_controls_account_actions_label">Account Actions</string>
   <string name="administrator_controls_log_out">Log Out</string>
+  <string name="log_out_dialog_cancel_button"><![CDATA[CANCEL]]></string>
```

Comment:
Replace this with 
`<string name="log_out_dialog_cancel_button">Cancel</string>
 `

Reply:
Done